File: B21-Final Otomasyon Sistemleri/Araba Kiralama/Araba Kiralama/KAYIT.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using System.Data.SqlClient;
using static System.Windows.Forms.VisualStyles.VisualStyleElement.ListView;

namespace Araba_Kiralama
{
    public partial class KAYIT : Form
    {
        private const string connectionString = "Server=DESKTOP-5BQNOPU\\SQLEXPRESS;Database=ArabaKiralamaDB;Integrated Security=true;";
        public KAYIT()
        {
            InitializeComponent();
        }
        private void btngeridon_Click(object sender, EventArgs e)
        {
            GİRİS giris = new GİRİS();
            giris.Show();
            this.Hide();
        }

        private void btnkayitol_Click(object sender, EventArgs e)
        {
            string bakiye = txtbakiye.Text;
            string ad = txtAd.Text;
            string soyad = txtSoyad.Text;
            string musteri_tc = txtTc.Text;
            string e_mail = txtEmail.Text;
            string sifre = txtSifre.Text;
            string telefon_no = txtTelefon.Text;
            string cinsiyet = cmbCinsiyet.SelectedItem.ToString();
            string ehliyet_no = txtEhliyetNo.Text;
            DateTime dogum_tarih = dtDogumTarihi.Value;

            using (SqlConnection connection = new SqlConnection(connectionString))
            {
                try
                {
                    connection.Open();

                    string query = "INSERT INTO musteri(ad,soyad,musteri_tc,e_mail,sifre,telefon_no,ehliyet_no,dogum_tarih,cinsiyet,bakiye) " +
                                  "VALUES (@ad, @soyad, @musteri_tc, @e_mail, @sifre, @telefon_no, @ehliyet_no, @dogum_tarih, @Cinsiyet, @bakiye)";
                    using (SqlCommand command = new SqlCommand(query, connection))
                    {
                    command.Parameters.AddWithValue("@bakiye",bakiye);
                    command.Parameters.AddWithValue("@ad", ad);
                    command.Parameters.AddWithValue("@soyad", soyad);
                    command.Parameters.AddWithValue("@musteri_tc", musteri_tc);
                    command.Parameters.AddWithValue("@e_mail", e_mail);
                    command.Parameters.AddWithValue("@sifre", sifre);
                    command.Parameters.AddWithValue("@telefon_no", telefon_no);
                    command.Parameters.AddWithValue("@ehliyet_no", ehliyet_no);
                    command.Parameters.AddWithValue("@dogum_tarih", dogum_tarih);
                    command.Parameters.AddWithValue("@Cinsiyet",cinsiyet);

                        int rowsAffected = command.ExecuteNonQuery();

                        if(rowsAffected > 0)
                        {
                            MessageBox.Show("Müşteri başarıyla kaydedildi.");
                        }
                        else
                        {
                            MessageBox.Show("Müşteri kaydedilirken bir hata oluştu.");
                        }
                    }
                    

                }
                catch (Exception ex)
                {
                    MessageBox.Show("Hata:  "+ ex.Message);
                }
            }
        }
    }
}

File: B21-Final Otomasyon Sistemleri/EmlakOtomasyonu/EmlakOtomasyonu/KAYIT.vb
Imports System.Data.SqlClient

Public Class KAYIT
    Private Sub Temizle()
        txtAd.Text = ""
        txtSoyad.Text = ""
        txtEmail.Text = ""
        txtSifre.Text = ""
        txtEmail.Text = ""
        txtTc.Text = ""
        txtTelefon.Text = ""
        cmbCinsiyet.SelectedItem.ToString()
        dtDogum.Value = DateTime.Now
    End Sub

    Private Sub btnGeridon_Click(sender As Object, e As EventArgs) Handles btnGeridon.Click
        Dim giris As New Form1()
        giris.Show()
        Me.Hide()
    End Sub

    Private baglanti As New SqlConnection("Data Source=DESKTOP-5BQNOPU\SQLEXPRESS;Initial Catalog=EmlakDB;Integrated Security=True")

    Private Sub btnKayitol_Click(sender As Object, e As EventArgs) Handles btnKayitol.Click
        baglanti.Open()
        Dim komut As New SqlCommand("INSERT INTO musteri(ad, soyad, tc_kimlik, telefon_no, e_mail, sifre, dogum_tarihi, cinsiyet) VALUES (@ad, @soyad, @tc_kimlik, @sifre, @e_mail, @telefon_no, @dogum_tarihi, @cinsiyet)", baglanti)
        komut.Parameters.AddWithValue("@ad", txtAd.Text)
        komut.Parameters.AddWithValue("@soyad", txtSoyad.Text)
        komut.Parameters.AddWithValue("@e_mail", txtEmail.Text)
        komut.Parameters.AddWithValue("@sifre", txtSifre.Text)
        komut.Parameters.AddWithValue("@tc_kimlik", txtTc.Text)
        komut.Parameters.AddWithValue("@telefon_no", txtTelefon.Text)
        komut.Parameters.AddWithValue("@dogum_tarihi", dtDogum.Value)
        komut.Parameters.AddWithValue("@cinsiyet", cmbCinsiyet.Text)
        Try
            Dim kontrol As Integer = komut.ExecuteNonQuery()

            If kontrol > 0 Then
                MessageBox.Show("Kayıt başarıyla eklendi.", "Bilgi", MessageBoxButtons.OK, MessageBoxIcon.Information)
                Temizle()
            Else
                MessageBox.Show("Kayıt eklenirken bir hata oluştu.", "Hata", MessageBoxButtons.OK, MessageBoxIcon.Error)
            End If
        Catch ex As Exception
            MessageBox.Show("Hata:" & ex.Message, "Hata", MessageBoxButtons.OK, MessageBoxIcon.Error)
        Finally
            baglanti.Close()
        End Try
    End Sub
End Class